.github/workflows/build_appliances.yaml

```
@@ -12,9 +12,19 @@ jobs:
       run: |
         git clone https://www.github.com/bdunne/imagefactory.git /build/gha/$GITHUB_RUN_ID/imagefactory
   build_appliances:
+    strategy:
+      max-parallel: 2
+      matrix:
+        target: [azure, ec2, gce, hyperv, libvirt, openstack, ovirt, vagrant, vsphere]
     runs-on: self-hosted
     needs: setup
+    env:
+      TARGET_PLATFORM: ${{ matrix.target }}
     steps:
     - uses: actions/checkout@v3
     - name: Build appliance images
-      run: "bin/nightly-build.sh"
+      env:
+        IBM_CLOUD_ACCESS_KEY: ${{ secrets.IBM_CLOUD_ACCESS_KEY }}
+        IBM_CLOUD_ENDPOINT: ${{ secrets.IBM_CLOUD_ENDPOINT }}
+        IBM_CLOUD_SECRET_KEY: ${{ secrets.IBM_CLOUD_SECRET_KEY }}
+      run: "ruby scripts/vmbuild.rb --type nightly --reference $GITHUB_REF_NAME --copy-dir $GITHUB_REF_NAME --only $TARGET_PLATFORM --upload --delete | tee -a /build/logs/${GITHUB_REF_NAME}_$(date +'%Y%m%d')_${GITHUB_RUN_ID}.log"
```

Very minor, but it might be preferable to do the following similar to how we do above in the clone step.  This also avoids escaping `"`

```suggestion
      run: |
        ruby scripts/vmbuild.rb --type nightly --reference $GITHUB_REF_NAME --copy-dir $GITHUB_REF_NAME --only $TARGET_PLATFORM --upload --delete | tee -a /build/logs/${GITHUB_REF_NAME}_$(date +'%Y%m%d')_${GITHUB_RUN_ID}.log
```

---

Done